src/RuntimeSettings.php

```diff
@@ -64,6 +71,11 @@ public function isDev()
      */
     public function getValue($name, $default = null)
     {
+        $url = $this->getUrlParam($name);
```

Comment:
Would it have been possible to just array_merge to two inputs into a single array?